scripts/03506-Adjoah.py

```diff
@@ -0,0 +1,8 @@
+# declaring variables
+
+name='Adwoa Serwaa Boafo'
+my_id='03506'
+email_add='adwoaserwaaboafo@gmail.com'
+
+#print statement
+print("Hello World, this is ",name + " with "+ my_id +" using Python for stage 2 task "+ email_add)
```

Comment:
Please make sure your output text conforms with this
`Hello World, this is [full name] with HNGi7 ID [ID] using [language] for stage 2 task [email_address]`
 Please look at the details

Reply:
i have updated the output now please